tools/pipelines/test-real-service.yml

```diff
@@ -167,8 +167,6 @@ stages:
       timeoutInMinutes: 360
       continueOnError: true
       testCommand: test:realsvc:odsp:report
-      stageVariables:
```

Comment:
in theory this should be fine, but let's do it in a different PR so there's less to revert if things go wrong :)

Reply:
..or I guess same PR is ok, but then you'd probably be on the hook for any issues if you don't want this whole change reverted.